### ISSUE TITLE
orientation issue

### ISSUE THREAD

**Issue by miradnan** [kartik-v/yii2-widget-datetimepicker#51]
I recently updated this yii2 extension and started having issues with orientation. 

When I create the DateTimePicker and click on it, the datetime popup opens up in bottom position. Not in the top position. Even when I am on he bottom of the page. It hides behind the window. It should open up at the top position instead.

```php
 echo DateTimePicker::widget([
                'model' => $model,
                'attribute' => 'date',
                'options' => ['placeholder' => 'Date...'],
                'convertFormat' => true,
                'pluginOptions' => [
                    'orientation' => 'top',
                    'autoclose' => true,
                    'format' => Yii::$app->formatter->datetimeFormat,
                    'startDate' => date(Yii::$app->formatter->dateFormat, strtotime('today')),
                    'todayHighlight' => true
                ]
 ]);
```

Here is an image showing the popup issue
![screen shot 2016-01-15 at 7 05 33 pm](https://cloud.githubusercontent.com/assets/397415/12354302/08e4d912-bbbb-11e5-8fdd-5042e5199217.png)

Experienced on Chrome browser

The same issue occurs on the demos you have listed. Also, the orientation option under pluginOptions does not have any effect.

**Comment by kartik-v** [comment]
Its an update to source plugin that does not probably have this feature any more and needs to be checked with [the plugin](https://github.com/smalot/bootstrap-datetimepicker) author. 

Will update if I find an option.

**Comment by kartik-v** [comment]
Update: The plugin now has an option called `pickerPosition` will update.

**Comment by kartik-v** [comment]
Resolved via updates to release v1.4.3
